saleor/order/emails.py

```diff
@@ -5,52 +5,78 @@
 from templated_email import send_templated_mail
 
 from ..core.utils import build_absolute_uri
+from .emails_helpers import get_order_confirmation_markup
```

Comment:
`emails_helpers` contains only functions related to the "email markup" feature, so maybe a better name would be `email_markup.py` plus a module docstring explaining that it's related to the Google's email markup feature? 
On the other hand, we have functions that generate `schema.org` markup in few other places, maybe we should group them into one app?

Reply:
Grouped as /schema/ app